doc/connection-to-the-internet-of-things/introduction.md

```diff
@@ -24,14 +24,15 @@ it is depicted in the following picture: 
 
 **FIWARE IoT Device Management GE architecture**
 
-This simple scenario considers only two Generic Enablers
+This simple scenario has got two basics
 implementations:
 
 -   **Orion ContextBroker:** That remains as the main front-end for
     developers. Developers access IoT data as attributes of entities
     representing devices and Developers may also send commands to
     devices by updating command-related attributes, providing they have
     access rights for that operation.
+    You can see the lastest version [here](https://github.com/Fiware/context.Orion/tree/release/1.7.0)
```

Comment:
-1 to this change. You are linking to a particular release of the Orion Broker, but what would happen when a new release is available? Are you going to change this link forever?